oci/table_oci_functions_function.go

```diff
@@ -0,0 +1,350 @@
+				Description: "Timeout for executions of the function. Value in seconds.",
+				Type:        proto.ColumnType_INT,
+			},
+
+			// json fields
+			{
+				Name:        "config",
+				Description: "The function configuration. Overrides application configuration.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getFunction,
+			},
+			{
+				Name:        "trace_config",
+				Description: "The trace configuration of the function.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// tags
+			{
+				Name:        "defined_tags",
+				Description: ColumnDescriptionDefinedTags,
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "freeform_tags",
+				Description: ColumnDescriptionFreefromTags,
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Standard Steampipe columns
+			{
+				Name:        "tags",
+				Description: ColumnDescriptionTags,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.From(functionTags),
+			},
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DisplayName"),
+			},
+
+			// Standard OCI columns
+			{
+				Name:        "region",
+				Description: ColumnDescriptionRegion,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Id").Transform(ociRegionName),
+			},
+			{
+				Name:        "compartment_id",
+				Description: ColumnDescriptionCompartment,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("CompartmentId"),
+			},
+			{
+				Name:        "tenant_id",
+				Description: ColumnDescriptionTenant,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     plugin.HydrateFunc(getTenantId).WithCache(),
+				Transform:   transform.FromValue(),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listFunctions(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	region := plugin.GetMatrixItem(ctx)[matrixKeyRegion].(string)
+	logger.Trace("listFunctions", "OCI_REGION", region)
+
+	equalQuals := d.KeyColumnQuals
+	var applicationId string
+
+	if equalQuals["application_id"] != nil {
+		applicationId = *types.String(equalQuals["application_id"].GetStringValue())
+	} else {
+		applicationId = *h.Item.(functions.ApplicationSummary).Id
+	}
+
+	// handle empty application id in list call
+	if applicationId == "" {
+		return nil, nil
+	}
+
+	// Create Session
+	session, err := functionsManagementService(ctx, d, region)
+	if err != nil {
+		logger.Error("listFunctions", "error_functionsManagementService", err)
+		return nil, err
+	}
+	
+	// Build request parameters
+	request := buildFunctionsFilters(equalQuals)
+	request.ApplicationId = &applicationId
+	request.Limit = types.Int(50)
+	request.RequestMetadata = common.RequestMetadata{
+		RetryPolicy: getDefaultRetryPolicy(),
+	}
+
+	limit := d.QueryContext.Limit
+	if d.QueryContext.Limit != nil {
```

Comment:
Can we set the minimum limit as per api documentation and update the logic bit for handle the minimum limit.